crates/bevy_input/src/input.rs

```diff
@@ -29,6 +29,35 @@ use bevy_ecs::schedule::State;
 /// * Using [`Input::clear_just_pressed`] or [`Input::clear_just_released`] instead.
 /// * Calling [`Input::clear`] or [`Input::reset`] immediately after the state change.
 ///
+/// ## Performance
+///
+/// For all operations, the following conventions are used:
+/// - **n** is the number of stored inputs.
+/// - **m** is the number of elements of another collection involved in the operation.
+/// - **\***-suffix denotes an amortized cost.
```

Comment:
Given that this isn't used here, I think it should be cut from this table.

Reply:
Sorry I don't understand, I'm using all the symbols?
To be sure, this **\*** is for the * symbol.

Reply:
My bad, I see now. It was hard to follow in markdown.